democracy/views/section_comment.py

```diff
@@ -56,3 +58,74 @@ class SectionCommentViewSet(BaseCommentViewSet):
     model = SectionComment
     serializer_class = SectionCommentSerializer
     create_serializer_class = SectionCommentCreateSerializer
+
+
+class RootSectionCommentSerializer(SectionCommentSerializer):
+    hearing = serializers.SerializerMethodField()
+
+    class Meta(SectionCommentSerializer.Meta):
+        fields = SectionCommentSerializer.Meta.fields + ['hearing']
+
+    def get_hearing(self, section_comment):
+        return section_comment.section.hearing.id
+
+
+class CommentPagination(LimitOffsetPagination):
+    default_limit = 50
+
+
+class CommentFilter(filters.FilterSet):
+    hearing = django_filters.CharFilter(name='section__hearing__id')
+
+    class Meta:
+        model = SectionComment
+        fields = ['authorization_code', 'section', 'hearing']
+
+
+# root level (Section)Comment endpoint
+class CommentViewSet(SectionCommentViewSet):
+    serializer_class = RootSectionCommentSerializer
+    pagination_class = CommentPagination
+    filter_backends = (filters.DjangoFilterBackend,)
+    filter_class = CommentFilter
+
+    def get_serializer(self, *args, **kwargs):
+        serializer_class = kwargs.pop("serializer_class", None) or self.get_serializer_class()
+        return serializer_class(*args, **kwargs)
+
+    def get_comment_parent_id(self):
+        parent_field = self.get_serializer_class().Meta.model.parent_field
+        try:
+            return self.request.data[parent_field]
+        except KeyError:
+            return None
+
+    def get_comment_parent(self):
```

Comment:
Yeah and it could be simplified probably in other ways as well, but I think that can wait until the final API structure has been decided to avoid making changes back and forth